include/amc6821.h

```diff
@@ -146,7 +146,7 @@ class AMC6821
          * 
          * @param fanspinup_toggle 
          */
-        void enableFanSpinup(bool fanspinup_toggle);
+        void disableFanSpinup(bool fanspinup_toggle);
```

Comment:
what's the reasoning behind making the function "disable" rather than "enable"? I feel like enable makes more logical sense but you would be like "setting disable to true" which actually means the fan spin up is false